Doc/whatsnew/3.7.rst

```diff
@@ -93,6 +93,10 @@ New Modules
 Improved Modules
 ================
 
+* Added another argument *monetary* in :meth:`format_string` of :mod:`locale`.
```

Comment:
There should be a subsection header before this line for the locale module (the modules with enhancements are listed in this section in alphabetical order).